### ISSUE TITLE
[Accessibility]: Non-code file view: File actions menu not read out to screen readers

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#35852]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34432

### Problem description

<img width="363" alt="image" src="https://user-images.githubusercontent.com/9516420/169797468-1638a87f-9a87-4500-8c91-5e60a63ab18f.png">

Screen readers cannot understand what this button does

### Expected behavior

Should have a label that describes usage

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
